backends/dpdk/backend.cpp

```diff
@@ -120,6 +121,7 @@ void DpdkBackend::convert(const IR::ToplevelBlock *tlb) {
         new DpdkAsmOptimization,
         new CollectUsedMetadataField(used_fields),
         new RemoveUnusedMetadataFields(used_fields),
+        new ValidateTableKeys()
```

Comment:
Added a new pass to emit error if the table keys from metadata struct along with the holes in the metadata structure do not fir in 64 bytes